doc/poolset/poolset.5.md

```diff
@@ -237,6 +237,10 @@ If the *SINGLEHDR* option is used, only the first part in each replica contains
 the pool part internal metadata. In that case the effective size of a replica
 is the sum of sizes of all its part files decreased once by 4096 bytes.
 
+*SINGLEHDR* option in the remote poolset file requires
```

Comment:
I think this should be mentioned in the rpmem manpage, not the other way around. It's rpmem that depends on poolsets.